### PR TITLE
Add more docker stats

### PR DESCRIPTION
**Pull request by hvanoch**
This pull requests add extra stats for docker.
This includes
- container uptime
- container size
- container filesystem 
- Totals per status

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by hvanoch** [comment]
Tests are failing. I believe the reasoning for the exception is that I have test for both v1 and v2 which probably include the same source file. not sure on how to get rid of that one.

**Comment by Jellyfrog** [comment]
I think its because its includes docker.inc.php twice,

but this functions should be generic instead, and it probably exists one that already can do the work in some helper class
```
In docker.inc.php line 11:
                                                                               
  Cannot redeclare convertToBytes() (previously declared in /home/runner/work  
  /librenms/librenms/includes/polling/applications/docker.inc.php:11)      
```

**Comment by hvanoch** [comment]
I moved the function to a separate helper class. Tests are also fixed so this MR is ready for review again.

**Comment by Jellyfrog** [comment]
Would you mind moving your new class into a function in one of the existing helper classes? Number.php would be a good match I suppose. Also add a comment for the function what it does

**Comment by hvanoch** [comment]
I have moved the conversion function to Number.php class as requested.

**Comment by hvanoch** [comment]
@Jellyfrog are you able to give it an other review?
Thanks

File: includes/polling/applications/docker.inc.php [hunk 1]
```diff
@@ -37,33 +22,73 @@ function convertToBytes(string $from): ?int
     return;
 }
 
+$version = intval($version);
+
+if ($version == 1) {
+    $output = 'LEGACY';
```
Comment (Jellyfrog) [inline includes/polling/applications/docker.inc.php:28]: @murrant is this how we usually do this?

Comment (hvanoch) [reply includes/polling/applications/docker.inc.php:28]: I took the dhcp stats as a reference: https://github.com/librenms/librenms/blob/master/includes/polling/applications/dhcp-stats.inc.php#L29

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**